spec/grape-active_model_serializers/error_formatter_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'spec_helper'
+require 'grape-active_model_serializers/error_formatter'
+
+describe Grape::ErrorFormatter::ActiveModelSerializers do
+  subject { Grape::ErrorFormatter::ActiveModelSerializers }
+  let(:backtrace) { ['Line:1'] }
+  let(:options) { Hash.new }
+  let(:env) { { 'api.endpoint' => app.endpoints.first } }
+  let(:original_exception) { StandardError.new('oh noes!') }
+
+  let(:app) { Class.new(Grape::API) }
+
+  before do
+    ActiveModel::Serializer.config.adapter = :json_api
```

Comment:
This pollutes future tests, so should either be set for all tests or unset in `after`.